examples_tests/41.VisibilityBuffer/occlusionCullingShaderCommon.glsl

```diff
@@ -0,0 +1,58 @@
+//TODO: rename
+//TODO: move buffers, that are not used by multiple shaders
+
+#include <nbl/builtin/glsl/utils/indirect_commands.glsl>
+
+#ifdef ENABLE_CUBE_COMMAND_BUFFER
```

Comment:
maybe instead of having an extra explicit `ENABLE_*` define, just check if `*_SET` was defined